test/metabase/models/setting/cache_test.clj

```diff
@@ -122,3 +124,28 @@
     ;; detect a cache out-of-date situation and flush the cache as appropriate, giving us the updated value when we
     ;; call! :wow:
     (setting-test/toucan-name)))
+
+;; sets site locale setting
+(expect
+  "fr"
+  (let [original-locale (java.util.Locale/getDefault)
```

Comment:
Can you put the code that restores the default locale in a `try-finally` block? That way if the tests throw an Exception we don't end up running other tests in French which could lead to hard-to-debug test failures

Reply:
sure thing!